src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.WorkerThread.cs

```diff
@@ -71,6 +71,31 @@ private static void WorkerThreadStart()
                             CultureInfo.CurrentCulture = CultureInfo.InstalledUICulture;
                             CultureInfo.CurrentUICulture = CultureInfo.InstalledUICulture;
```

Comment:
BTW: Where is the equivalent code that resets culture and priority in the CoreCLR thread pool? I am not able to find it.